app/views/home/_find_growers.html.haml

```diff
@@ -0,0 +1,8 @@
+.find-growers.home-banner
+
+  #placesmap
```

Comment:
@Skud : I think you mentioned wanting this map to update as you typed something into the input. That shouldn't be too hard to do. Should I implement now or break out into another story?